tasks/demos.tex

```diff
@@ -16,13 +16,28 @@
+
+
+\paragraph*{Task activity:}
+ 
+Within the actual Binder capability, we will demonstrate following two reproducible research configurations.  We demonstrate these research use caseses and show barriers that has been preventing these workflow to be pulished as reproducible science.   
+
+%These information will give first feed back to \WPref{reproducibility}, \WPref{impact} 
+%ll enrich the process of propose better accesible optimised 
+%research workflow that can benefit researchers themselves, but also include reproducible aspects. 
+
+
   \begin{compactitem}
   \item FAIR Nordic Earth System Modelling: this science demonstrator leverages Binder@HOME (model development, education, single column or very simple model configuration), Binder@HPC (operational runs at scale including on EuroHPC), data publishing (publication of simulation results from blue-sky research);
-  \item Marine physical and fish habitats modelling and analysis: In marine research there are well established reproducible research examples such as Argopy \TODO{ref to https://dx.doi.org/10.21105/joss.02425}  or pangeo ecosystems (http://gallery.pangeo.io/repos/pangeo-gallery/physical-oceanography/). Still in Ifremer, we have number of researchers who depends on commercial software for their data analysis and does not have access to publish reproducible research workflows.  We demonstrate these research use caseses and study their barriers and propose better accesible optimised research workflow that can benefit researchers themselves, but also include reproducible aspects.   The techinical tool kit for this activity will be co-developped with WP2 and WP3. We'll demonstrate these use cases on Binder@home and Binder@HPC. Ifremer is in charge of managing numerous marine databases and information systems and the HPC system have direct access to these datasets.  We will optimise the access to these datasets, and will study the possibility to extend that with the usage from cloud infrastructure, in connected efforts with Binder running on Clouds, such as EOSC. 
-This effort and outcome of it will connect to the Digital Twin Ocean project to allow ocean data and models relevant to biodiversity to be re-used by researchers and engineers. This will provide a concept demonstration of ingesting ocean data and model output that can be reproduced through the existing ocean research infrastructures.; 
+  \item Demonstration of marine physics and fish habitats modelling and analysis using Pangeo ecosystem.  
+  
+\TODO{Tina, move this to section 1? but where??
+This effort and outcome of it will connect to the Digital Twin Ocean project to allow ocean data and models relevant to biodiversity to be re-used by researchers and engineers. This will provide a concept demonstration of ingesting ocean data and model output that can be reproduced through the existing ocean research infrastructures
```

Comment:
Probably section 1.2.7 on Science Applications.

Reply:
thanks, I keep the todo as is now and after the merge, I'll move it to 1.2.4 (ex 1.2.7) .